src/components/InventoryDetail/ApplicationDetails.js

```diff
@@ -14,11 +14,15 @@ const ApplicationDetails = ({ onTabSelect, appList, ...props }) => {
     const history = useHistory();
     const dispatch = useDispatch();
     const searchParams = new URLSearchParams(search);
-    const items = useSelector(({ entityDetails }) => entityDetails?.activeApps);
+    // eslint-disable-next-line max-len
+    const items = useSelector(({ entityDetails }) => entityDetails?.activeApps || []).filter(({ isVisible }) => isVisible !== false);
```

Comment:
Might be a better to wrap after `filter`
```suggestion
    const items = useSelector(({ entityDetails }) => entityDetails?.activeApps || [])
    .filter(({ isVisible }) => isVisible !== false);
```